level-2/README.md

```diff
@@ -0,0 +1,27 @@
+# karma-clone
+## Level 2
+The aim of this exercise is to add a new section to the homepage, and create a new page with an Order form.
+
+### Set up
+
+You can choose to add your code for Level 2 to the same branch you were using for Level 1, or to create a new 'level-2' branch. 
+
+# Instructions
+
+- You'll find the designs inside the 'level 2' folder.
+- Add the new design block in your `index.html`
+- Create a new page called `order.html` and re-create the design for the Order form page (find this in the Level 2 folder as well)
+- For the Order page styles, you can keep using the same `style.css` file and add your Order page styles in there. Make sure the stylesheet is imported in the `<head>` of `order.html`.
```

Comment:
```suggestion
- For the order page styles, you can keep using the same `style.css` file and add your order page styles in there.
```
WDYT about leaving out this hint so that students can try to recall the solution themselves?